x-pack/solutions/observability/plugins/apm/server/routes/transactions/get_transaction/index.ts

```diff
@@ -58,7 +63,15 @@ export async function getTransaction({
     TRANSACTION_TYPE,
   ] as const);
 
-  const optionalFields = asMutableArray([PROCESSOR_NAME, SERVICE_LANGUAGE_NAME] as const);
+  const optionalFields = asMutableArray([
```

Comment:
These errors keep happening ever since we adapted the code to support otel, could we add a test to validate whether these fields are returned in the response?

Reply:
That's a great idea, thanks! As this is a blocker for some users I would do it in a separate PR: I added an issue: [#206947](https://github.com/elastic/kibana/issues/206947)